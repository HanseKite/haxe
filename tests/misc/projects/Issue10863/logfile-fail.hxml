Any
-D messages-log-file=/blargh
