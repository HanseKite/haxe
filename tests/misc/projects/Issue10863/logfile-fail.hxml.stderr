Error opening log file: /blargh: Permission denied. Logging to file disabled (-D messages_log_file)
